### DISCUSSION TITLE
What genmon features are supported with a generac powerpact model?

### DISCUSSION THREAD

**Discussion by Nate52441** [jgyates/genmon#1209] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).
- [X] I have included the generator make and model in my submission.

### Version

Powerpact (evo) pre install so no genmon version 

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have not submitted logs

### What is/are your question(s)?

I got a new generac powerpact installed and was looking at this awesome project to supplement what generac failed to do. I came across a discussion on the powerpact not supporting generator voltage and was wondering what exactly is supported by genmon with a powerpact generator? 

I’m not trying to do anything too complex, I just want to setup alerts for the following and want to know if I can do it. 

notify when generator is running 
Notify on low main batt voltage (bad battery)
Notify on power outage 
Notify on failure to start generator 

thanks so much for any assistance to community can provide!

**Comment by skipfire** [comment, marked as answer]
If you setup notifications you will get all 4 of those scenarios plus others.  The low battery I believe would fire at the point it creates an alarm on the controller, though frequently a battery that has to be replaced will still read an acceptable voltage but just not have the strength to start the motor.  That is one of the things that a weekly test should help eliminate as a concern though.

**Reply by Nate52441** [reply]
Ahhh I wasn’t sure if it’d really know that the generator didn’t start if power pact doesn’t support outputting the generator voltage through the accessory port. Do you know if genmon with powerpact will support reading utility voltage?

**Reply by jgyates** [reply]
yes. Utility voltage is exported by the power pact controller. The generator output voltage and service hours counters are not exported. Any alarm conditions are reported via notifications so that should cover most failures and outage conditions. The controller will generator an alarm if the battery is out of range or has a charging issue.

**Reply by Nate52441** [reply]
Thanks for the info, you folks are awesome!
